### ISSUE TITLE
Determining reference word or phrase

### ISSUE THREAD

**Issue by scottpeterson** [gryphonshafer/Quizzing-Rule-Book#26]
Reference Question: "Determining reference word or phrase."

Because of the imprecise way this is worded (it is not defined), there is no way for a Quizmaster to apply it. 

Either it should be objectively defined, or it should be removed.

If the question on the card if “if a man is what?” and the quizzer gives “a man is what?” or “if a man what?” , there is no way for the quizmaster to know if either contain the “determining word or phrase?”

I see three options for improving this rule:

1. If you really want the quizzer to have to guess on 50/50 reference questions (the actual origin for this rule), JUST WRITE THAT. Define 50/50 reference questions. Then say the quizzer has to guess the right one first. Then we don’t need this ambiguous language which has to be interpreted for non 50/50 reference questions.
2. Switch to my preferred handling of quizzer provided reference questions. ( if it’s same type and if sum of information in question and answer is the same, accept it).
3. Switch to CMD’s handling of quizzer provided reference questions. (quizzer has to ultimately get to the verbatim question. If they give a question that is same type, with same sum of information, they can keep providing questions.

**Comment by scottpeterson** [comment]
If the question on the card if “if a man is what?” and the quizzer gives “a man is what?” or “if a man what?”  how am I supposed to know if either contain the “determining word or phrase?”

I see three options for improving this rule:

1. If you really want the quizzer to have to guess on 50/50 reference questions (the actual origin for this rule), JUST WRITE THAT.   Then we don’t need this ambiguous language which has to be interpreted for non 50/50 reference questions. 
2. Switch to my preferred handling of quizzer provided reference questions. ( if it’s same type and if sum of information in question and answer is the same, accept it). 
3. Switch to CMD’s handling of quizzer provided reference questions.   (quizzer has to ultimately get to the verbatim question.  If they give a question that is same type, with same sum of information, they can keep providing questions.

**Comment by ARMediting** [comment]
I agree that "determining word or phrase" has no clear meaning, but the example below it of "lawless deeds" indicates that the wrong guess on a 50/50 results in an incorrect answer. But the example you gave of "if a man is what?" is trickier. It fits in the space between the two examples ("King of the what?" vs "King of what?" and "Lawless what?" vs "What deeds?").

I do see a fourth option, however:

4. Ban "pure" 50/50s, where both words of a phrase could be used as separate questions. There aren't that many. It would be like banning FTVs that start with the same five words. 

But that doesn't solve the addition or subtraction of "little" words. It's similar to the issue of how obscure a phrase to use as a reference (we'd never write "Of the what?" for example, even though it could be perfectly valid). I'm intrigued by CMD's solution, which keeps the need for the exact question but gives the quizzer space to find it.

**Comment by jswingle** [comment]
From day one I've been saying with reference questions that my priority is having a clear and understandable rule. Even though calling quizzers wrong on 50/50 references wasn't my preferred solution, I would have been OK with it if we at least had gotten a clear and understandable rule out of it. This rule is not at all understandable. So as far as I'm concerned, this was a negative change for the rulebook. We have an equally non-understandable rule, with the one exception that it's super clear now that we _have_ to count quizzers wrong on 50/50 references. It needs to be revised.

I really like CMD's way of handling references. I do think that it makes it a bit tougher on the officials, though. You have to check several questions for validity and keep all the questions given by the quizzer in your head as you deliberate on a ruling. I would be totally fine with this being the rule.

Ultimately, I like option (2). For all the following reasons:

- Clearly understandable to the quizzer
- Easy to handle for officials
- More or less objective
- Simply makes the most sense. "lawless what?" and "what deeds?" _aren't_ fundamentally different questions, they're two sides of the same coin. When I write INT questions for a question set, I don't _typically_ write BOTH "Who so loved the world?" AND "God so loved what?" in the same question set, because I consider them very similar questions. So if the quizzer gives the "wrong" one on a 50/50, I don't think their question is quite so different than many others think.

Regarding @ARMediting 's option (4), I think this is actually harder to define than we may originally think. 50/50s go beyond just "lawless what?" vs. "what deeds?" It can also include longer phrases, such as for example "Who may be saved?" vs. "They may be what?" I also would prefer a solution that allows for us to ask these otherwise good questions, rather than banning them and adding more rules for question writers to keep in mind.

**Comment by kclimenhaga** [comment]
I agree that banning 50/50s is problematic. I do like the "sum of the information" wording most of the time, but occasionally it can become too specific. For example, there are numerous occurrences of phrases like "As it is written" or "For it is written". When I write a reference question with these, I write "What is written", because I think that is the best question. If a quizzer says "For it is written, what?" I don't want the quizzer to be called incorrect for adding the "for", as long as it is still the same question type. I'm not sure how to word this in a way that can be applied consistently, but I would like to try to find a way if possible.

**Comment by jswingle** [comment]
Practically speaking, in PNW we would always count a quizzer right for such a substantially similar "sum of the information" as the addition of the single word "for." I'm not sure how to define that in the rules though. This is another advantage of the CMD approach of allowing multiple questions to be given - it solves this problem perfectly.

On the other hand, since a quizzer _can_ technically narrow it down to the exact question, it could be seen as a "tough but fair" ruling to demand the exact wording.

**Comment by ARMediting** [comment]
For the record, I'm still most comfortable with option 1. I've never been mad at the rulebook for needing to guess on a 50/50. It's like jumping on "Therefore."
